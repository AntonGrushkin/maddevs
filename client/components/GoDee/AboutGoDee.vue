<template>
  <section id="about-godee" class="about-godee">
    <div class="container">
      <div class="about-godee__text-content">
        <h2 class="about-godee__main-title">About GoDee</h2>
        <p class="about-godee__description-bold">The app-based system allows users to find a route, <br> select the time, book a seat, and pay for the ride online.</p>
        <p class="about-godee__description">GoDee is a reliable way to commute in Ho Chi Minh city, Vietnam . Shuttle buses ride <br> along the express routes linking the major districts and popular destinations of the city. <br> It helps users to save their time, money, and planet due to less ecological impact.</p>
      </div>
      <div class="about-godee__people-image-wrapper">
        <img src="@/assets/img/GoDee/svg/about-godee-people.svg" class="about-godee__people-image" alt="People">
      </div>
    </div>
  </section>
</template>

<script>
export default {
  name: 'AboutGoDee'
};
</script>

<style lang="scss">
  @import '../../assets/styles/vars';
  @import '../../assets/styles/get-vw';

  .about-godee {
    margin-top: get-vw(325px);
    margin-bottom: get-vw(120px);
    background: url('../../assets/img/GoDee/svg/about-godee-bg.svg') no-repeat bottom;
    background-size: contain;

    &__text-content {
      text-align: center;
      margin-bottom: get-vw(45px);
    }

    &__main-title {
      margin-bottom: get-vw(23px);
      font-size: get-vw(90px);
      letter-spacing: get-vw(-3px);
      color: $accent-orange-text-color;
    }

    &__main-title,
    &__description-bold {
      font-family: 'Hoves-Bold';
    }

    &__description-bold,
    &__description {
      letter-spacing: get-vw(-1px);
    }

    &__description-bold {
      margin-bottom: get-vw(25px);
      font-size: get-vw(32px);
      letter-spacing: get-vw(-1px);
      color: $text-color--black;
    }

    &__description {
      font-size: get-vw(24px);
      line-height: get-vw(33px);
      font-family: 'Hoves-Regular';
      color: $text-color--grey-dark;
    }

    &__people-image-wrapper {
      width: 100%;
      display: flex;
      justify-content: flex-end;
    }

    &__people-image {
      width: get-vw(393px);
      height: get-vw(262px);
    }
  }
</style>
