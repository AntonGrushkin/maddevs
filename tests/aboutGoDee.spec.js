import {
  mount
} from '@vue/test-utils';
import AboutGoDee from '@/components/GoDee/AboutGoDee';

describe('About GoDee', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(AboutGoDee);
  });

  // ------ IMPORTANT ----- //
  test('is a Vue instance', () => {
    expect(wrapper.exists()).toBeTruthy();
  });

  test('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
  // --------------------- //
});
