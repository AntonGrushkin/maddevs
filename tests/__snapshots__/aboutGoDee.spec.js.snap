// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About GoDee renders correctly 1`] = `
<section
  class="about-godee"
  id="about-godee"
>
  <div
    class="container"
  >
    <div
      class="about-godee__text-content"
    >
      <h2
        class="about-godee__main-title"
      >
        About GoDee
      </h2>
       
      <p
        class="about-godee__description-bold"
      >
        The app-based system allows users to find a route, 
        <br />
         select the time, book a seat, and pay for the ride online.
      </p>
       
      <p
        class="about-godee__description"
      >
        GoDee is a reliable way to commute in Ho Chi Minh city, Vietnam . Shuttle buses ride 
        <br />
         along the express routes linking the major districts and popular destinations of the city. 
        <br />
         It helps users to save their time, money, and planet due to less ecological impact.
      </p>
    </div>
     
    <div
      class="about-godee__people-image-wrapper"
    >
      <img
        alt="People"
        class="about-godee__people-image"
        src="@/assets/img/GoDee/svg/about-godee-people.svg"
      />
    </div>
  </div>
</section>
`;
